src/drawbridge.rs

```diff
@@ -267,11 +273,54 @@ pub fn login(
             .set_password(secret)
             .context("Failed to save user credentials")?;
     }
+
```

Comment:
Please avoid making unrelated diffs like this

Reply:
You mean the extra blank line, or the refactoring to a generalized function?